src/compute-pr-actions.ts

```diff
@@ -375,7 +375,9 @@ function makeStaleness(now: string, author: string, otherOwners: string[]) { //
         const state = days <= freshDays ? "fresh" : days <= attnDays ? "attention" : days <= nearDays ? "nearly" : "done";
         const kindAndState = `${kind}:${state}`;
         const explanation = Comments.StalenessExplanations[kindAndState];
-        const comment = Comments.StalenessComment(author, otherOwners)[kindAndState];
+        const expires = new Date(now);
+        expires.setDate(expires.getDate() + nearDays);
+        const comment = Comments.StalenessComment(author, otherOwners, expires)[kindAndState];
```

Comment:
1. Maybe do the formatting here, so `comments` is still simple?
2. I don't like much the chance of using a date >= the days in the month, but I'm guessing that it's well-defined?
3. But maybe the `moment` library can simplify this?